### PR TITLE
docs: upgrade to `.yml` based issue-templates

### PR DESCRIPTION
**Pull request by antazoey**
### What I did

* Upgrades to .yml based issue templates
* Offers bigger section for reproduction steps

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Review by bitwise-constructs — DISMISSED** [review]
Great idea to make the template yml

**Comment by antazoey** [comment]
im gonna convert the others too

**Review by fubuloubu — DISMISSED** [review]

**Comment by fubuloubu** [comment]
> im gonna convert the others too

I wonder if github has a way to make this org-level instead or each repo

**Comment by antazoey** [comment]
> I wonder if github has a way to make this org-level instead of each repo

there are some differences though, like `eip712` doesn't care about your `ape-config` file or ape plugins, etc.
and `ape-vyper` may wonder about your vyper version + syntax

**Comment by antazoey** [comment]
i jush wish i can view / demo these before merging

**Comment by fubuloubu** [comment]
> > I wonder if github has a way to make this org-level instead of each repo
> 
> there are some differences though, like `eip712` doesn't care about your `ape-config` file or ape plugins, etc. and `ape-vyper` may wonder about your vyper version + syntax

yeah exactly, set an org-level, and then set specific ones for repos that don't match

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - type: input
+    attributes:
+      label: What command(s) is the bug in?
+      description: Leave empty if not relevant
+      placeholder: "For example: ape test"
+  - type: dropdown
+    attributes:
+      label: Operating System
+      description: What operating system are you on?
+      options:
```
Comment (fubuloubu) [inline .github/ISSUE_TEMPLATE/bug.yml:30]: is there a way to disallow `None` as an option?

File: .github/ISSUE_TEMPLATE/work-item.yml
Comment (fubuloubu) [inline .github/ISSUE_TEMPLATE/work-item.yml:1]: tbh I don't feel like we really use this one anymore

**Comment by fubuloubu** [comment]
> > im gonna convert the others too
> 
> I wonder if github has a way to make this org-level instead of each repo

yes there is!
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (antazoey) [reply .github/ISSUE_TEMPLATE/bug.yml:30]: Done. Had to add:

```
validations:
      required: true
```

File: .github/ISSUE_TEMPLATE/work-item.yml
Comment (antazoey) [reply .github/ISSUE_TEMPLATE/work-item.yml:1]: removed.

**Comment by antazoey** [comment]
> yes there is! https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types

OK, I followed those instructions and created https://github.com/ApeWorX/.github
Then, I removed any file here that was un-altered, `FUNDING.yml` and `PULL_REQUEST_TEMPLATE.yml`

File: SECURITY.md
Comment (fubuloubu) [inline SECURITY.md:1]: why was this file deleted? probably needs an overhaul, but I don't think it should get deleted

File: .github/FUNDING.yml
Comment (fubuloubu) [inline .github/FUNDING.yml:1]: definitely don't delete this

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (fubuloubu) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Do we want to get rid of this?

**Review by fubuloubu — APPROVED** [review]

**Comment by fubuloubu** [comment]
This gave me some other ideas for the `.github` repo

**Comment by fubuloubu** [comment]
https://github.com/ApeWorX/.github/pull/1
